### PR TITLE
append_to_stream rename

### PR DESCRIPTION
**Pull request by mpraglowski**

### CODE REVIEW

File: railseventstore.org/source/docs/exceptions.html.md.erb [hunk 1]
```diff
@@ -34,13 +34,21 @@ If you want to have an event present in multiple streams, you have to link it wi
 
 ### RubyEventStore::InvalidExpectedVersion    
 
+<% if version_above('0.30.0') %>
+Occurs when invalid `exception_version` is passed in `append`, `link_to_stream`, `publish_event` or `publish_events`.
```
Comment (joelvh) [inline railseventstore.org/source/docs/exceptions.html.md.erb]: Was `publish` rename merged?

Comment (mpraglowski) [reply railseventstore.org/source/docs/exceptions.html.md.erb]: https://github.com/RailsEventStore/rails_event_store/pull/377 yes - but I don't want to change docs in this PR - keep it focused on append_to_stream - however this needs update.

Comment (mpraglowski) [reply railseventstore.org/source/docs/exceptions.html.md.erb]: The changes in docs for #377 are here https://github.com/RailsEventStore/rails_event_store/pull/378

**Comment by paneq** [comment]
@mpraglowski what do you think about `persist` instead of `append` ? Imagine we start persisting commands as well without triggering handlers.

**Comment by mpraglowski** [comment]
Some said:

> I don't assume commands must to be stored in the same stream as as events for an object/entity/aggregate.

So why you assume we will use the same method to store events & commands ;)

**Comment by paneq** [comment]
> So why you assume we will use the same method to store events & commands ;)

I see no reason to have 2 different methods for it. For me it's about persisting a message in both cases.

**Comment by pawelpacana** [comment]
@mpraglowski would merge but conflicts

**Pull request merged**